addOns/reports/src/main/java/org/zaproxy/addon/reports/ReportDialog.java

```diff
@@ -231,10 +232,12 @@ public void changedUpdate(DocumentEvent e) {
         this.addCheckBoxField(TAB_SCOPE, FIELD_DISPLAY_REPORT, reportParam.isDisplayReport());
 
         Template defaultTemplate = extension.getTemplateByConfigName(reportParam.getTemplate());
+        List<String> templates = extension.getTemplateNames();
+        Collections.sort(templates);
```

Comment:
You could use SortedSet? 

<sub>Or we could target Java 11 since 9 introduced SortedList :)</sub>

Reply:
It needs to be a `List`.

Reply:
Yeah, core limitations :/